extensions/msal-node-extensions/package.json

```
@@ -45,7 +49,7 @@
   },
   "dependencies": {
     "@azure/msal-common": "^5.1.0",
-    "bindings": "git://github.com/samuelkubai/node-bindings.git#v1.6.0",
+    "bindings": "git+https://github.com/samuelkubai/node-bindings.git#v1.6.0",
```

I think this could also be this:
```suggestion
    "bindings": "samuelkubai/node-bindings#v1.6.0",
```

See: https://docs.npmjs.com/cli/v6/commands/npm-install

---

Yeah there's a few options, but the docs aren't clear on what protocol they use under the hood, I thought it's best to be explicit unless it's documented somewhere